packages/react-composites/src/composites/CallComposite/components/CapabilitiesNotficationBar.tsx

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* @conditional-compile-remove(capabilities) */
+import {
+  CapabilitiesChangeInfo,
+  CapabilitiesChangedReason,
+  ParticipantCapabilityName,
+  ParticipantRole
+} from '@azure/communication-calling';
+/* @conditional-compile-remove(capabilities) */
+import { IMessageBarProps, MessageBar, MessageBarType, Stack } from '@fluentui/react';
+/* @conditional-compile-remove(capabilities) */
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+/* @conditional-compile-remove(capabilities) */
+import { useLocale } from '../../localization';
+/* @conditional-compile-remove(capabilities) */
+import { CallCompositeStrings } from '../Strings';
+
+/* @conditional-compile-remove(capabilities) */
+/**
+ * @beta
+ */
+export interface CapabilitiesChangeNotificationBarProps extends IMessageBarProps {
+  capabilitiesChangeInfo?: CapabilitiesChangeInfo;
+  participantRole?: ParticipantRole;
+}
+
+/* @conditional-compile-remove(capabilities) */
+/**
+ * @beta
+ */
+export interface CapabalityChangeNotification {
+  /**
+   * Name of capability
+   */
+  capabilityName: ParticipantCapabilityName;
+  /**
+   * Is capability now available
+   */
+  isPresent: boolean;
+  /* @conditional-compile-remove(capabilities) */
+  /**
+   * Reason capability was changed
+   */
+  changedReason?: CapabilitiesChangedReason;
+  /* @conditional-compile-remove(capabilities) */
+  /**
+   * Role of participant when capability changed
+   */
+  role?: ParticipantRole;
+  /**
+   * The latest timestamp when this notification was observed.
+   *
+   * When available, this is used to track errors that have already been seen and dismissed
+   * by the user.
+   */
+  timestamp?: Date;
+}
+
+/* @conditional-compile-remove(capabilities) */
+interface DismissedNotification {
+  capabilityName: ParticipantCapabilityName;
+  dismissedAt: Date;
+  activeSince?: Date;
+}
+
+/* @conditional-compile-remove(capabilities) */
+/**
+ * Notification bar for capabilities changes
+ * @beta
+ */
+export const CapabilitiesChangeNotificationBar = (props: CapabilitiesChangeNotificationBarProps): JSX.Element => {
+  const locale = useLocale();
+
+  // Timestamp for when this component is first mounted.
+  // Never updated through the lifecycle of this component.
+  const mountTimestamp = useRef(new Date(Date.now()));
```

Comment:
Will the calling sdk always populate the capabilities before this mounts? Is there a chance we get into a (for example) lobby page _then_ the SDK populates the capabilities (i.e. goes from `undefined -> [capabilities]`) in which case would we be incorrectly showing a capability changed notification?

Reply:
Removed the mount timestamp.